stats/rpc_constants.textpb

```diff
@@ -0,0 +1,620 @@
+# Constants for collecting specific RPC stats, including:
+#   - Measurement Descriptor, which describes a data point (measurement) type.
+#   - View Descriptor, which specifies an AggregationDescriptor and a set of tag keys.
+#     View Descriptor has one of the two kinds of Aggregation types associated with the View,
+#     Interval Aggregation Descriptor or Distribution Aggregation Descriptor.
+#     Aggregations are broken down by the unique set of matching tag values for each measurement.
+
+# Census defined RPC client MeasurementDescriptors.
+measurement_descriptor {
+  name: "/rpc/client/error_count"
+  description: "Count of client RPCs that received an error (unsuccessful) response"
+  unit: {
+    numerators: SCALAR
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/client/request_bytes"
+  description: "Request bytes of client RPCs"
+  unit: {
+    numerators: BYTES
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/client/response_bytes"
+  description: "Response bytes of client RPCs"
+  unit: {
+    numerators: BYTES
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/client/roundtrip_latency"
+  description: "RPC roundtrip latency in milliseconds on client RPC side"
+  unit: {
+    power10: -3
+    numerators: SECONDS
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/client/server_elapsed_time"
+  description: "Server elapsed time in milliseconds on client RPC side"
+  unit: {
+    power10: -3
+    numerators: SECONDS
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/client/uncompressed_request_bytes"
+  description: "Uncompressed Request bytes of client RPCs"
+  unit: {
+    numerators: BYTES
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/client/uncompressed_response_bytes"
+  description: "Uncompressed Response bytes of client RPCs"
+  unit: {
+    numerators: BYTES
+  }
+}
+
+# Census defined RPC server MeasurementDescriptors.
+measurement_descriptor {
+  name: "/rpc/server/error_count"
+  description: "Count of server RPCs that received an error (unsuccessful) response"
+  unit: {
+    numerators: BYTES
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/server/request_bytes"
+  description: "Request bytes of server RPCs"
+  unit: {
+    numerators: BYTES
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/server/response_bytes"
+  description: "Response bytes of server RPCs"
+  unit: {
+    numerators: BYTES
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/server/server_elapsed_time"
+  description: "Server elapsed time in milliseconds on server RPC side"
+  unit: {
+    power10: -3
+    numerators: SECONDS
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/server/server_latency"
+  description: "Latency in milliseconds on server RPC side"
+  unit: {
+    power10: -3
+    numerators: SECONDS
+  }
+}
```

Comment:
If the difference is semantic, it makes sense to keep them and other languages should implement them as defined. If they are semantically the same but it's just C++ happens to have two implementations, they should be consolidated.